.github/workflows/validate_manifests.yml

```diff
@@ -0,0 +1,19 @@
+name: Validate yaml configurations
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  manifests-validation:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout test-infra ⤵️
+        uses: actions/checkout@v3
+
+      - name: Setup Kubeval ⛓️
+        uses: lra/setup-kubeval@v1.0.1
+        
+      - name: yaml config validation 🔍
+        run: kubeval --directories config/prow --ignore-missing-schemas
```

Comment:
Thanks @cappellinsamuele! Just missing a newline

Reply:
Thank you!